src/lib/Components/Artist/Artworks/index.tsx

```diff
@@ -19,15 +19,7 @@ interface RenderSectionParams {
   mapPropsToArtworksConnection: (Props) => any
 }
 
-interface Props extends ViewProperties {
-  artist: {
-    counts: {
-      for_sale_artworks: number
-      artworks: number
-    }
-    not_for_sale_artworks: any[]
-    for_sale_artworks: any[]
-  }
+interface Props extends RelayProps, ViewProperties {
   relay: any
```

Comment:
I don’t believe this prop is actually used in this module, but if it is then maybe pull in the appropriate typing from `react-relay`?